contracts/finance/PlushCoinWallets.sol

```diff
@@ -10,16 +10,15 @@ import "../PlushApps.sol";
 import "../token/ERC20/Plush.sol";
 
 contract PlushCoinWallets is Initializable, PausableUpgradeable, AccessControlUpgradeable, UUPSUpgradeable {
-
     bytes32 public constant PAUSER_ROLE = keccak256("PAUSER_ROLE");
     bytes32 public constant OPERATOR_ROLE = keccak256("OPERATOR_ROLE");
     bytes32 public constant UPGRADER_ROLE = keccak256("UPGRADER_ROLE");
 
-    Plush plush;
-    PlushApps plushApps;
+    Plush public plush;
+    PlushApps public plushApps;
 
-    uint256 minimumBet;
-    address plushFeeWallet;
+    uint256 public minimumBet;
```

Comment:
Maybe it's worth renaming because it does not quite reflect the meaning that it is used for the deposit function?